viewer.js

```diff
@@ -7,21 +7,21 @@ var twitch = window.Twitch.ext
 
 // create the request options for our Twitch API calls
 var requests = {
-  getParticipants: createRequest("GET", "query"),
+  getParticipants: createRequest("GET"),
 }
 
-function createRequest(type, method) {
+function createRequest(type) {
   return {
     type: type,
-    url: location.protocol + "//localhost:8081/participants/" + method,
+    url: "https://montano-twitch-extension.herokuapp.com/",
     success: renderTopParticipants,
     error: logError,
   }
 }
 
 function setAuth(token) {
   Object.keys(requests).forEach((req) => {
-    requests[req].headers = { Authorization: `Bearer ${token}` }
+    //requests[req].headers = { Authorization: `Bearer ${token}` }
```

Comment:
Did you leave the setAuth function because you plan to use it in the future?

I think if you don't use it, you can remove it.